JS.md

```diff
@@ -571,7 +555,7 @@ Return the result of [`CreateMemoryObject`](#creatememoryobject)(`memory`).
 
 ### CreateMemoryObject
 
-Given a [`Memory.memory`](https://github.com/WebAssembly/spec/blob/master/interpreter/spec/memory.mli#L1)
+Given a [memory instance](https://webassembly.github.io/spec/exec/runtime.html#memory-instances)
 `m`, to create a `WebAssembly.Memory`:
 
 Let `buffer` be a new `ArrayBuffer` whose
```

Comment:
Should this be changed to create a new `SharedArrayBuffer` depending on whether `m` is shared?  Or is that a separate proposal?